magenta/lib/melodies_lib.py

```diff
@@ -248,6 +248,15 @@ def _get_last_on_off_events(self):
         return (i, last_off)
     raise ValueError('No events in the stream')
 
+  def deepcopy(self):
```

Comment:
might as well also define __deecopy__ while you're here, in case someone wants to use copy.deepcopy